src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiErrorSchemaGenerator.cs

```diff
@@ -27,28 +27,36 @@ internal static class OpenApiErrorSchemaGenerator
         public static IDictionary<string, OpenApiSchema> CreateODataErrorSchemas(this ODataContext context)
         {
             Utils.CheckArgumentNull(context, nameof(context));
+            var rootNamespaceName = context.GetErrorNamespaceName();
 
             return new Dictionary<string, OpenApiSchema>()
             {
-                // odata.error
-                { "odata.error", CreateErrorSchema() },
-
-                // odata.error.main
-                { "odata.error.main", CreateErrorMainSchema() },
-
-                // odata.error.detail
-                { "odata.error.detail", CreateErrorDetailSchema() },
-
-                // odata.error.innererror
-                { "odata.error.innererror", CreateInnerErrorSchema(context) }
+                { $"{rootNamespaceName}odataerror", CreateErrorSchema(rootNamespaceName) },
```

Comment:
Can we define as `const` these string values that have been used in more than one place so that they are easily re-usable within the doc.?

Reply:
do you mean the odataerror part?

Reply:
we should probably upper camel case these so we get proper class names in the code-gen too.

Reply:
> do you mean the odataerror part?

Yes, and the `mainerror`, `errordetails`, etc...

Reply:
thanks, solved